lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -300,8 +299,8 @@ def track_lines(
     subpixel minimum and angle between the successive normal vectors is computed. The candidate
     with the lowest score is then selected.
 
-    If desired (`refine=True`), each spot is subsequently refined by performing a bias-corrected
-    centroid optimization according to [2]_.
+    Each spot is subsequently refined by performing a bias-corrected centroid optimization
+    according to [2]_.
```

Comment:
I think the information here can be combined into line 289. As is, it seems a bit repetitive